_posts/2022-05-16-zarr-2.12.0a1-release.markdown

```diff
@@ -6,4 +6,76 @@ categories: blog
+
+The old implementation iterated through all the `old` chunks and removed those that didn’t exist in the `new` chunks. As a result, it led to significant time delays when appending data to Zarr arrays in cloud services like S3.
+
+The new and improved implementation will iterate through each dimension and only find and remove the chunk slices in `old` but not in `new` data. It also introduced a mutable list to dynamically adjust the number of chunks along the already-processed dimensions to avoid duplicate chunk removal.
+
+This improvement was added by [hailiangzhang](https://github.com/hailiangzhang) with PR [#1014](https://github.com/zarr-developers/zarr-python/pull/1014).
+
+## Other enhancements
+
+- If you have created a fsspec filesystem outside of Zarr, you can now pass it as a keyword argument to `FSStore`.
+
+This feature was added by [Ryan Abernathy](https://github.com/rabernat) with PR [#911](https://github.com/zarr-developers/zarr-python/pull/911).
+
+- Added number encoder for `json.dumps` to support NumPy integers in `chunks` arguments. 
+
+This enhancement was added by [Eric Prostate](https://github.com/ericpre) with PR [#933](https://github.com/zarr-developers/zarr-python/pull/933) and the issue was raised by [Mark Dickinson](https://github.com/mdickinson) with [#697](https://github.com/zarr-developers/zarr-python/issues/697).
+
+## Bugs, Documentation and Maintenance
+
+Fix bug that made it impossible to create an FSStore on unlistable filesystems (e.g. some HTTP servers) by [Ryan Abernathey](https://github.com/rabernat); [#993](https://github.com/zarr-developers/zarr-python/issues/993).
+
+Update resize doc to clarify surprising behaviour by [hailiangzhang](https://github.com/hailiangzhang); [#1022](https://github.com/zarr-developers/zarr-python/pull/1022).
+
+Pre-commit configuration now includes `YAML` check by [Shivank Chaudhary](https://github.com/Alt-Shivam); [#1015](https://github.com/zarr-developers/zarr-python/issues/1015) & [#1016](https://github.com/zarr-developers/zarr-python/issues/1016).
+
+## More information
+
+Details on these features as well as the full list of all changes in `2.12.0a1` are available on the release notes. Check [here](https://zarr--1023.org.readthedocs.build/en/1023/release.html#a1).
+
+## Appreciation 🙌🏻
+
+Before the pre-release version `2.12.0a1` there were releases `2.11.1`, `2.11.2`, & `2.11.3` from Zarr Python package. A special shout-out to all the contributors who made previous releases possible:
+
+- [Tobias Kölling](https://github.com/d70-t)
+- [Ben Jeffrey](https://github.com/benjeffery)
+- [Vyas Ramasubramani](https://github.com/vyasr)
+- [Tom White](https://github.com/tomwhite)
+
+Also, a huge thanks to the contributors who made the current version `2.12.0a1` possible! 🙌🏻
+
+If you find the above features useful and end up using them, please mention [@zarr_dev](https://twitter.com/zarr_dev) on Twitter and tweet using #ZarrData, and we’ll make sure to get it featured! ✌🏻
```

Comment:
I could see using the twitter link from https://zarr.dev on the tag.

Reply:
Done.